Box.V2/Config/Constants.cs

```diff
@@ -160,6 +160,8 @@ public static class Constants
         public const string AuthHeaderKey = "Authorization";
         public const string V1AuthString = "BoxAuth api_key={0}&auth_token={1}";
         public const string V2AuthString = "Bearer {0}";
+        public const string BearerTokenType = "bearer";
+        public const int AccessTokenExpirationTime = 3600; // seconds
```

Comment:
Maybe this should be called `ClientCredentialsAccessTokenExpirationTime` ?

Reply:
The expiration time value is tied directly to the OAuth, not to the CCG itself - JWT Auth also has the same expiration time.